src/System.Net.Primitives/tests/FunctionalTests/CredentialCacheTest.cs

```diff
@@ -57,7 +62,6 @@ private static CredentialCacheCount CreateUriCredentialCacheCount(CredentialCach
 
             cc.Add(uriPrefix2, authenticationType1, credential3); count++;
             cc.Add(uriPrefix2, authenticationType2, credential4); count++;
-
```

Comment:
In previous comment, David asked me to delete it...and I submitted a new commit for that